docs/cni/kubernetes/vagrant-coreos/VagrantCoreOS.md

```diff
@@ -82,51 +82,36 @@ And finally check that Docker is running on both hosts by running
 
     docker ps
 
-## 2. Using your cluster
+## 2. Configuring the Cluster 
+### 2.1 Configure `kubectl`
+Let's configure `kubectl` so you can access the cluster from your local machine. Make sure you have `kubectl` installed locally.  The version you choose depends on your host OS.
 
-### 2.1 Deploying SkyDNS
-You now have a basic Kubernetes cluster deployed using Calico networking.  Most Kubernetes deployments use SkyDNS for Kubernetes service discovery.  The following steps configure the SkyDNS service.
-
-Log on to the master node.
-```
-vagrant ssh k8s-master 
-```
-
-Deploy the SkyDNS application using the provided Kubernetes manifest.
+For Mac:
 ```
-kubectl create -f skydns.yaml
+wget http://storage.googleapis.com/kubernetes-release/release/v1.3.5/bin/darwin/amd64/kubectl
```

Comment:
The master-config.yaml we use for this demo uses 1.4.0-alpha.3, not 1.3.5.

Please also check through the rest of the PR to make sure the versions match where they need to.